quiche/src/recovery/gcongestion/bbr2/startup.rs

```diff
@@ -66,21 +67,36 @@ impl ModeImpl for Startup {
 
         let has_bandwidth_growth =
             self.model.has_bandwidth_growth(congestion_event, params);
+        // TODO: Check full_bandwidth_reached to determine if exit due to
+        // bandwidth plateau
 
-        #[allow(clippy::absurd_extreme_comparisons)]
-        if params.max_startup_queue_rounds > 0 && !has_bandwidth_growth {
+        let check_persistent_queuing =
+            params.max_startup_queue_rounds > 0 && !has_bandwidth_growth;
+        if check_persistent_queuing {
             // 1.75 is less than the 2x CWND gain, but substantially more than
             // 1.25x, the minimum bandwidth increase expected during
             // STARTUP.
-            self.model.check_persistent_queue(1.75, params);
-        }
+            let PersistentQueueOutcome {
+                full_bandwidth_reached,
+                rounds_with_queueing: _,
+            } = self.model.check_persistent_queue(1.75, params);
+
+            // TODO: Exit due to persistent queue
+            let _exit_due_to_persistent_queue = full_bandwidth_reached;
```

Comment:
The approach in https://github.com/cloudflare/quiche/pull/2057 has evolved a bit.  I would consider not exposing full_bandwidth_reached in PersistentQueueOutcome yet.  And remove the TODOs associated with 2057.

It would be good to have a version of 2057 rebased to take advantage of changes in this PR.